### DISCUSSION TITLE
Introduction and doubt/question

### DISCUSSION THREAD

**Discussion by fjherna** [dgtaheno/artillery-Genius#24] [General]
Hi All,
[NOTE TO ALL: I am really sorry that my first post was in Spanish (my mother-language). :( ]

Hi all and in particularly to dgtaheno,

Sorry for approaching you like this without having participated before. I have seen a project on GitHub, in which I am very interested and I think it is related to this forum.

I have an Artillery Genius (basic and one of the first since I have had it for a few years now; I seem to remember it was in January 2021).

I admit that I have quite abandoned it, and just now I have decided to take it up again.

I'll give you a little background on what my current situation is. I'm totally new to 3D printing and have only made a few pieces of my own, apart from printing some from thingiverse.
:)
The most I have done is install Marlin 2.0.5.3 and then compile the current Marlin 2.1.x myself to update it, so I changed the original extruder for a "Trianglelab MATRIX" one (similar to the HEMERA, but cheaper).

A few days ago I considered looking for a complementary screen to manage the printer and the Firmware a little better if possible.

In my search I found some interesting information on YouTube (https://www.youtube.com/watch?v=H3mHDbtcaxU). I then thought about purchasing a FYSETC screen, but the problem is that in that article it only mentions Marlin 1.9.0 and 2.0.5.3. Furthermore, it is from three years ago and is currently not active. I asked about the possibility of using a Marlin 2.1.x with those FYSETC or similar screens, but I don't expect anyone to answer.

In view of this, I continued searching and yesterday I found your project in which Firmware 2.1.2.1 is mentioned and some BigtreeTech TFT Touchscreen screens that seemed to me could have possibilities with my Artillery Genius.
In the links I have seen (https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw) the project is quite recent.

Well, I don't want to go into more detail (perhaps it's already too much and I'm sorry), because my intention was only to make contact with you regarding this project, and to find out if it is still going.

Of course I offer to collaborate in whatever is necessary, always considering the modest measure of my possibilities and limitations.

Everyone receive a cordial greeting,

fjherna1

**Comment by dgtaheno** [comment]
Hi @fjherna ,

I think it is the same answered in [https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/issues/23#issuecomment-1986811898](https://github.com/dgtaheno/artillery-Genius-Genius_Pro-X1-Marlinfw/issues/23#issuecomment-1986811898).

Best regards.
